calico_node/tests/st/utils/docker_host.py

```diff
@@ -556,8 +556,25 @@ def create_network(self, name, driver="calico", ipam_driver="calico-ipam",
         :param subnet: The subnet IP pool to assign IPs from.
         :return: A DockerNetwork object.
         """
-        nw = DockerNetwork(self, name, driver=driver, ipam_driver=ipam_driver,
-                           subnet=subnet)
+        class DummyNetwork(object):
```

Comment:
I think it would be nicer to put this class in network.py, as then it will be easier for people to see that it needs to remain consistent with DockerNetwork.